packages/react-components/src/templates/__tests__/ManuscriptForm.test.tsx

```diff
@@ -635,7 +683,15 @@ it('resets form fields to default values when no longer visible', async () => {
     }),
   ).not.toBeInTheDocument();
 
-  userEvent.click(screen.getByRole('button', { name: /Submit/i }));
+  const submitButton = screen.getByRole('button', { name: /Submit/i });
+
+  await waitFor(() => {
+    expect(submitButton).toBeEnabled();
+  });
+
+  expect(submitButton).toBeEnabled();
```

Comment:
We can remove this code. It was necessary when we were updating a file and needed to wait for the upload to complete to enable the submit button. However, since we're now providing the file as props, this is no longer required.